### PR TITLE
Add ability to drag and drop elements

### PR DESCRIPTION
**Pull request by rthompson-campaignmonitor**
# Description

Add ability to drag and drop elements to target elements.

### CODE REVIEW

File: SmartWebDriver/WebBrowser.cs [hunk 1]
```diff
@@ -742,6 +742,34 @@ public void RightClick(IWebElement webElement, string description)
             }
         }
 
+        public void DragAndDrop(PageElement elementToDrag, PageElement targetElement)
```
Comment (sn3akypete) [inline SmartWebDriver/WebBrowser.cs:749]: Please add a summary description to this method

**Review by sn3akypete — APPROVED** [review]
1 Minor comment then LGTM to merge. Nice one!

File: SmartWebDriver/WebBrowser.cs (hunk 1 above)
Comment (rthompson-campaignmonitor) [reply SmartWebDriver/WebBrowser.cs:749]: I won't lie, I am new to GitHub. For adding the summary description, do I need to commit that summary and create a new pull request, or can I update this one?

Edit: nevermind, I believe I figured it out on accident :)

Comment (sn3akypete) [reply SmartWebDriver/WebBrowser.cs:749]: Nice, thanks for that. This is now good to merge, I unfortunately no longer have Write access, so I'm not able to at the moment, I'm hoping to get permission back. But in the meantime, if you have write access, feel free to merge!

Comment (rthompson-campaignmonitor) [reply SmartWebDriver/WebBrowser.cs:749]: Looks like I do not have write access. Do you know who I should contact to get this merged in?

**Pull request merged**